src/tests/ftest/security/pool_security_groups.yaml

```
@@ -5,24 +5,15 @@ hosts:
     - server-A
   test_clients:
     - server-B
-  transport_config:
-    - allow_insecure: False
-    - ca_cert: ~/.daos/daosCA.crt
-    - cert: ~/.daos/daos_agent.crt
-    - key: ~/.daos/daos_agent.key
-    - server_name: server
-timeout: 600
+timeout: 1200
 server_config:
   name: daos_server
   port: 10001
-daos_server:
-  start:
-     insecure: False
 pool_acl:
-  scm_size: 134217728
+  scm_size: 1334217728
```

Was a `3` accidentally inserted in this value? If not, why is the `scm_size` being increased here?

```suggestion
  scm_size: 134217728
```

---

Change the scm_size due to origin scm_size 134217728 too small and causing failure.

---

Is turning on security actually making the SCM usage higher?  In the most recently fully run weekly test, this test passed all of it's tests:

https://build.hpdd.intel.com/job/daos-stack/job/daos/job/weekly-testing/86/testReport/(root)/DaosRunPoolSecurityTest/

So are you sure any failure you are seeing is related to SCM size?

How as the new value chosen?  It appears to be just an insertion of a `3` into the existing value increasing it by 9.94x.  Surely something more scientific should be done to choose a reasonable new value, yes?

Also, please don't resolve conversations that you didn't start.  The Resolve button is for the person who raised the issue to acknowledge that the issue has been resolved to their satisfaction.

---

From the failure, the min scm_size is about 126M.

---

Any idea why this was working as recently as https://build.hpdd.intel.com/job/daos-stack/job/daos/job/weekly-testing/86/testReport/(root)/DaosRunPoolSecurityTest/ then?